src/SkprConfig.php

```diff
@@ -49,15 +49,15 @@ public function load(string $filename = self::DEFAULT_FILENAME, string $symlink
     }
     // We cache the config in memory.
     if (empty($this->config)) {
-      $data = @file_get_contents($filename);
+      $data = @file_get_contents(realpath($filename));
       // If the data is not found, symlinks may be outdated.
       if ($data === FALSE) {
```

Comment:
Can't this just be an `if (!file_exists(realpath($filename)))` rather than trying to get the contents twice?

Reply:
If we get the contents we just jump to parsing the json and thats it. If the file doesn't exist, we clear caches and try again.